### ISSUE TITLE
Honk in city sound is too loud

### ISSUE THREAD

**Issue by jnns** [rafaelmardojai/blanket#205]
I'd like to get other people's opinion on the city sounds (`city.ogg`).

**Comment by rafaelmardojai** [comment]
CC @Porrumentzio, what do you think?

**Comment by Porrumentzio** [comment]
I'll take a look on that as soon as I can. If I don't get back in four days, call me again.

**Comment by Porrumentzio** [comment]
I don't think that this is annoying. This time the graphic representation is not so useful, as we don't perceive short loud sounds as loud as if they would last for more time. We can compress the clip, but very probably it won't change a lot the perception for what I said before (the spontaneousity of the sound will remain there, even if more quiet).

Is that horn annoying for you both?

**Comment by rafaelmardojai** [comment]
> Is that horn annoying for you both?

Hmm, never heard that sound for a long time, but now I realize that the honk becomes annoying quickly...

**Comment by dino8890** [comment]
Hi, this is messing with me as well, it is so sudden that it becomes distracting and pulls you out of whatever you're focusing on.
